ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -694,6 +694,64 @@ literals:
   - PRESENT
   - ABSENT
 
+# Added for TPKD day2
+- condensing_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- condensing_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- oil_heater_status:
+  - ON
+  - OFF
+- inhibit_start_time_sensor
+- load_limit_capacity
+- total_compressor_count
+- compressor_run_time_sensor
+- guidevane_percentage_command
+- oil_temperature_sensor
+- oil_pump_differential_pressure_sensor
+- exhaust_gas_compressor_temperature_sensor
+- bearing_temperature_sensor
+- winding_temperature_sensor
+- compressor_temperature_setpoint
```

Comment:
Is there an associated `compressor_temperature_sensor`?

Reply:
added compressor_temperature_sensor